lib/glific/wa_managed_phones.ex

```diff
@@ -183,4 +184,48 @@ defmodule Glific.WAManagedPhones do
   @spec has_any_phones(list()) :: {:ok, list()} | {:error, String.t()}
   defp has_any_phones([]), do: {:error, "No active phones available"}
   defp has_any_phones(wa_managed_phones), do: {:ok, wa_managed_phones}
+
+  @doc """
+  add the phone status
+  """
+  @spec status(String.t(), non_neg_integer()) :: {:ok, WAManagedPhone} | {:error, String.t()}
+  def status(new_status, phone_id) do
+    organization_id = Repo.get_organization_id()
+
+    phone =
+      from(p in WAManagedPhone,
+        where: p.phone_id == ^phone_id
+      )
+      |> Repo.one()
```

Comment:
Can we use the existing function `get_wa_managed_phone` (non-raise function, we have to create one) or `list_wa_managed_phones`.